src/type/index.ts

```diff
@@ -10,8 +10,8 @@ export interface ServerlessErrorConstructor {
   readonly prototype: ServerlessError;
 }
 
-export interface ServerlessClasses extends Serverless {
-  classes?: { Error: ServerlessErrorConstructor };
+export type ServerlessClasses = Serverless & {
+  classes?: { Error: any };
```

Comment:
Do we have a reason for this change?

Reply:
Fair call, no valid reason. There was an issue with the types when the packages were upgraded. I'll push a fix up